src/ai/backend/client/cli/ssh.py

```diff
@@ -12,14 +12,20 @@
 from .pretty import print_fail, print_info
 
 
+def find_backendai_executable():
+    if __debug__ and os.path.isfile("./backend.ai"):
+        return "./backend.ai"
+    return "backend.ai"
+
+
 @contextlib.contextmanager
 def container_ssh_ctx(session_ref: str, port: int) -> Iterator[Path]:
     random_id = secrets.token_hex(16)
     key_filename = "id_container"
     key_path = Path(f"~/.ssh/id_{random_id}").expanduser()
     try:
         subprocess.run(
-            ["backend.ai", "session", "download", session_ref, key_filename],
+            [find_backendai_executable(), "session", "download", session_ref, key_filename],
```

Comment:
```suggestion
            [*find_backendai_executable(), "session", "download", session_ref, key_filename],
```
Since `find_backendai_executable()` returns a list!